cli.js

```diff
@@ -96,7 +103,7 @@ api.run()
 		logger.exit(api.failCount > 0 || api.rejectionCount > 0 || api.exceptionCount > 0 ? 1 : 0);
 	})
 	.catch(function (err) {
-		if (err instanceof Error) {
+		if (err.name === 'Error') {
```

Comment:
Previous code was wrong.

`err instanceof Error` is always true, because `TypeError`, `SyntaxError`, etc all inherit from `Error`. So when there was a `TypeError`, we only showed `err.message` without stack.